### PR TITLE
DOC-2761 and DOC-2417: Add info about Programs page to learner dashboard docs

### PR DESCRIPTION
**Pull request by srpearce**
[DOC-2761](https://openedx.atlassian.net/browse/DOC-2761) and [DOC-2417](https://openedx.atlassian.net/browse/DOC-2822)

This PR adds information about the new **Programs** page on the learner dashboard to both course author and learner docs.
- [x] Tech review: @schenedx (Simon, please let me know if someone else should TR this)
- [ ] Product review: @jsubramanian or @dabdul-hathi (not sure who to tag while Marco is on vacation)
- [ ] Doc review (copy edit): @lamagnifica @catong @pdesjardins

### CODE REVIEW

File: en_us/links/links.rst [hunk 1]
```diff
@@ -16,8 +16,6 @@
 
 .. _Open edX Analytics wiki: http://edx-wiki.atlassian.net/wiki/display/OA/Open+edX+Analytics+Home
 
-.. _openedx-analytics: http://groups.google.com/forum/#!forum/openedx-analytics
-
```
Comment (lamagnifica) [inline en_us/links/links.rst:20]: ?

File: en_us/shared/getting_started/dashboard_settings_profile.rst [hunk 2]
```diff
@@ -4,7 +4,7 @@
 Exploring Your Dashboard, Settings, and Profile
 ###################################################
 
-This topic describes the edX dashboard, your account settings, and your
+This topic describes the edX learner dashboard, your account settings, and your
```
Comment (lamagnifica) [inline en_us/shared/getting_started/dashboard_settings_profile.rst]: maybe strike "edX" here?

File: en_us/shared/getting_started/dashboard_settings_profile.rst [hunk 3]
```diff
@@ -47,23 +47,29 @@ When you create an edX account or enroll in an edX course, you automatically
...
- :alt: The learner dashboard with two current courses and two archived courses
-     listed.
+Your dashboard has two pages.
+
+* The **Courses** page lists all the courses you have enrolled in, whether the
+  course is current or complete. For more information, see :ref:`Courses Page`.
+
+* The **Programs** page lists any programs you are enrolled in. If you
+  are enrolled in any course that is part of a program, that program
+  is visible on this page. For more information, see :ref:`Programs Page`.
```
Comment (lamagnifica) [inline en_us/shared/getting_started/dashboard_settings_profile.rst]: is visible --> appears (or is included)

is this course page/program page info true for all sites, or only for edx.org/Edge? perhaps it should be under an only:: Partners?

File: en_us/shared/getting_started/dashboard_settings_profile.rst [hunk 4]
```diff
@@ -47,23 +47,29 @@ When you create an edX account or enroll in an edX course, you automatically
 go to your dashboard. You can also access your dashboard at any time by
 selecting your username in the upper right corner of any edX page.
 
-.. image:: ../../../shared/images/SFD_Dashboard.png
- :width: 500
- :alt: The learner dashboard with two current courses and two archived courses
-     listed.
+Your dashboard has two pages.
```
Comment (lamagnifica) [inline en_us/shared/getting_started/dashboard_settings_profile.rst]: two --> the following

File: en_us/shared/getting_started/dashboard_settings_profile.rst [hunk 5]
```diff
@@ -47,23 +47,29 @@ When you create an edX account or enroll in an edX course, you automatically
...
+
+* The **Courses** page lists all the courses you have enrolled in, whether the
```
Comment (lamagnifica) [inline en_us/shared/getting_started/dashboard_settings_profile.rst]: have --> are

File: en_us/shared/getting_started/dashboard_settings_profile.rst [hunk 6]
```diff
@@ -47,23 +47,29 @@ When you create an edX account or enroll in an edX course, you automatically
...
 
-.. note - The current image is a Photoshopped combination of the old and new
-.. dashboards, so it's not 100% accurate. It's good enough for now, but I plan
-.. to update it when the new dashboard releases.
+.. _Courses Page:
 
-Your dashboard contains important information about your courses. You can take
-several actions on your dashboard.
+=================
+The Courses Page
```
Comment (lamagnifica) [inline en_us/shared/getting_started/dashboard_settings_profile.rst]: could this heading/topic be reframed in terms of action the learner would want to take, a task they need to complete, a problem they can solve? 

Reviewing and Updating Your Course Enrollments
Using the Courses Page to Access a Course
Accessing Courses and Course Information 
etc.

File: en_us/shared/getting_started/dashboard_settings_profile.rst [hunk 7]
```diff
@@ -47,23 +47,29 @@ When you create an edX account or enroll in an edX course, you automatically
...
+=================
+
+The **Courses** page opens by default. This page contains important information
```
Comment (lamagnifica) [inline en_us/shared/getting_started/dashboard_settings_profile.rst]: when does it open?

File: en_us/shared/getting_started/dashboard_settings_profile.rst [hunk 8]
```diff
@@ -47,23 +47,29 @@ When you create an edX account or enroll in an edX course, you automatically
 go to your dashboard. You can also access your dashboard at any time by
```
Comment (lamagnifica) [inline en_us/shared/getting_started/dashboard_settings_profile.rst]: other edits could be made above, like on line 23 add "learner" to "dashboard" for consistency

File: en_us/shared/getting_started/dashboard_settings_profile.rst [hunk 9]
```diff
@@ -47,23 +47,29 @@ When you create an edX account or enroll in an edX course, you automatically
...
 selecting your username in the upper right corner of any edX page.
 
-.. image:: ../../../shared/images/SFD_Dashboard.png
```
Comment (lamagnifica) [inline en_us/shared/getting_started/dashboard_settings_profile.rst:50]: delete this image file if not used in any other guide

File: en_us/shared/getting_started/dashboard_settings_profile.rst [hunk 10]
```diff
@@ -47,23 +47,29 @@ When you create an edX account or enroll in an edX course, you automatically
...
+The **Courses** page opens by default. This page contains important information
+about your courses. You can take several actions on the **Courses** page.
 
 * Access your current and archived courses.
 
   To access a current or archived course, select **View Course** or **View
-  Archived Course**. Note that archived courses may provide limited access to
-  content.
+  Archived Course** next to the name of the course. Note that archived courses
+  may not provide access to all content.
```
Comment (lamagnifica) [inline en_us/shared/getting_started/dashboard_settings_profile.rst]: may --> might (or "do not always"

File: en_us/shared/getting_started/dashboard_settings_profile.rst [hunk 11]
```diff
@@ -76,24 +82,81 @@ several actions on your dashboard.
 * Change your course email settings.
 
   #. Select the Options icon for the course, and then select **Course emails**.
```
Comment (lamagnifica) [inline en_us/shared/getting_started/dashboard_settings_profile.rst:96]: @clrux it looks like we might be calling this the action-action-more button in the DOM. Could we possibly add a tooltip for this icon? It would be good to be consistent and have a way to surface what we mean in the UI.

File: en_us/shared/getting_started/dashboard_settings_profile.rst [hunk 12]
```diff
@@ -76,24 +82,81 @@ several actions on your dashboard.
...
   To unenroll, select the Options icon for the course, select **Unenroll**, and
-  then select **Unenroll** in the confirmation dialog box.
+  then select **Unenroll** in the confirmation dialog box. The Options icon
+  resembles a gear and appears next to **View Course** or **View Archived
+  Course**.
+
+     .. image:: ../../../shared/images/LearnDash_GearIcon.png
+       :width: 125
+       :alt: The Options icon next to the View Course button on the learner
+           dashboard.
```
Comment (lamagnifica) [inline en_us/shared/getting_started/dashboard_settings_profile.rst]: I think you can strike the image here rather than showing it twice in such close proximity.

File: en_us/shared/getting_started/dashboard_settings_profile.rst [hunk 13]
```diff
@@ -76,24 +82,81 @@ several actions on your dashboard.
...
 
 * View your grades in courses that you have completed.
 
   Your final grade appears under the name of the course.
 
-* Download a certificate.
+* Access a certificate.
```
Comment (lamagnifica) [inline en_us/shared/getting_started/dashboard_settings_profile.rst]: why the change from download to access here, but not in lines 114-115?

File: en_us/shared/getting_started/dashboard_settings_profile.rst [hunk 14]
```diff
@@ -76,24 +82,81 @@ several actions on your dashboard.
...
 
   To download a certificate, select **Download Your Certificate (PDF)** or
   **Download Your ID Verified Certificate (PDF)**. You can then print the PDF
   certificate.
 
+* Find out whether your ID verification is still active.
```
Comment (lamagnifica) [inline en_us/shared/getting_started/dashboard_settings_profile.rst]: maybe xref to info about what to do if it is not (or to find out what this means)

File: en_us/shared/getting_started/dashboard_settings_profile.rst [hunk 15]
```diff
@@ -76,24 +82,81 @@ several actions on your dashboard.
...
+
+  If you have enrolled in the verified certificate track for a course, your ID verification status appears together with the course image for that course.
```
Comment (lamagnifica) [inline en_us/shared/getting_started/dashboard_settings_profile.rst]: strike together

File: en_us/shared/getting_started/dashboard_settings_profile.rst [hunk 16]
```diff
@@ -76,24 +82,81 @@ several actions on your dashboard.
...
+
+.. _Programs Page:
+
+=================
+The Programs Page
+=================
```
Comment (lamagnifica) [inline en_us/shared/getting_started/dashboard_settings_profile.rst]: same comment, can this be recast to provide a solution to a problem or answer to a question?

File: en_us/shared/getting_started/dashboard_settings_profile.rst [hunk 17]
```diff
@@ -76,24 +82,81 @@ several actions on your dashboard.
...
+
+The **Programs** page lists all of your programs. You see a
+program on this page if you are enrolled in any courses that are part of that
+program.
+
+EdX currently offers XSeries programs, which are groups of two to ten courses
```
Comment (lamagnifica) [inline en_us/shared/getting_started/dashboard_settings_profile.rst]: make this an only for partners?

File: en_us/shared/getting_started/dashboard_settings_profile.rst [hunk 18]
```diff
@@ -76,24 +82,81 @@ several actions on your dashboard.
...
+that explore a specific subject in depth. You can earn a certificate for the
+XSeries program and for the individual courses in the program. For more
+information about XSeries programs, see `XSeries Programs`_.
+
+On the **Programs** page, you can perform the following actions.
+
+* Access information about a program and enroll in more of the program's
+  courses.
+
+  To access information about a program, select that program. A page opens and
```
Comment (lamagnifica) [inline en_us/shared/getting_started/dashboard_settings_profile.rst]: and provides --> with

File: en_us/shared/getting_started/dashboard_settings_profile.rst [hunk 19]
```diff
@@ -76,24 +82,81 @@ several actions on your dashboard.
...
+  provides more information about the program, as well as links to enroll in
+  courses in that program.
+
+* View your progress through the program.
+
+  Under the name of the program are two indications of your progress through
+  the program. A message specifies the number of courses for which you have
+  earned certificates in the program. Additionally, a colored bar indicates the
+  number of program courses that you have enrolled in and completed.
```
Comment (lamagnifica) [inline en_us/shared/getting_started/dashboard_settings_profile.rst]: strike "enrolled in and"?

File: en_us/shared/getting_started/dashboard_settings_profile.rst [hunk 20]
```diff
@@ -76,24 +82,81 @@ several actions on your dashboard.
 * Change your course email settings.
 
   #. Select the Options icon for the course, and then select **Course emails**.
+     The Options icon resembles a gear and appears next to **View Course** or
+     **View Archived Course**.
+
+     .. image:: ../../../shared/images/LearnDash_GearIcon.png
+       :width: 125
+       :alt: The Options icon next to the View Course button on the learner
+           dashboard.
```
Comment (lamagnifica) [inline en_us/shared/getting_started/dashboard_settings_profile.rst:103]: in the image, I didn't realize that the pink circle was indicating the icon -- I thought it was part of the page! maybe use the squoval shape instead, or add an arrow pointing to the pinnk circle?

**Comment by lamagnifica** [comment]
some suggestions about presentation, and a nagging suspicion that we don't actually have a good, consistent name for that freakin' icon throughout the system -- DOM, tooltip, doc. Use what makes sense.
👍

File: en_us/shared/getting_started/dashboard_settings_profile.rst [hunk 21]
```diff
@@ -47,23 +47,29 @@ When you create an edX account or enroll in an edX course, you automatically
...
-.. image:: ../../../shared/images/SFD_Dashboard.png
- :width: 500
- :alt: The learner dashboard with two current courses and two archived courses
-     listed.
+Your dashboard has two pages.
+
+* The **Courses** page lists all the courses you have enrolled in, whether the
+  course is current or complete. For more information, see :ref:`Courses Page`.
+
+* The **Programs** page lists any programs you are enrolled in. If you
```
Comment (schenedx) [inline en_us/shared/getting_started/dashboard_settings_profile.rst]: The word `enroll` here is not accurate IMO. How about something like `lists any programs you can work to complete` or something along that line?

File: en_us/shared/getting_started/dashboard_settings_profile.rst (hunk 3 above)
Comment (schenedx) [inline en_us/shared/getting_started/dashboard_settings_profile.rst]: The functionality described here is true for all sites, but currently only turned on for edx.org/edge.

File: en_us/shared/getting_started/dashboard_settings_profile.rst (hunk 7 above)
Comment (schenedx) [inline en_us/shared/getting_started/dashboard_settings_profile.rst]: I don't like the clause `opens by default`. How about it is simply `the default page learn sees after they logged in?`

File: en_us/shared/getting_started/dashboard_settings_profile.rst (hunk 13 above)
Comment (schenedx) [inline en_us/shared/getting_started/dashboard_settings_profile.rst]: Maybe we want to make a distinction here that this certificate is a `course certificate`? Because after user completes a program/xseries, they will get a program certificate. However, that is only on edx.org. I don't know if it will be appropriate to mention to openedx community?

File: en_us/shared/getting_started/dashboard_settings_profile.rst (hunk 18 above)
Comment (schenedx) [inline en_us/shared/getting_started/dashboard_settings_profile.rst]: I don't know `A page opens...` is a good description of the action here. It makes me think that I will be opening a new browser window. Maybe I am wrong?

File: en_us/shared/getting_started/dashboard_settings_profile.rst [hunk 22]
```diff
@@ -76,24 +82,81 @@ several actions on your dashboard.
...
+* Access information about a program and enroll in more of the program's
+  courses.
+
+  To access information about a program, select that program. A page opens and
+  provides more information about the program, as well as links to enroll in
+  courses in that program.
+
+* View your progress through the program.
+
+  Under the name of the program are two indications of your progress through
```
Comment (schenedx) [inline en_us/shared/getting_started/dashboard_settings_profile.rst]: I thought there are 3 indications. 1 Green color bar for completed course, a yellow bar for courses currently enrolled in and a gray bar for courses within the program the student haven't enrolled in.

File: en_us/links/links.rst (hunk 1 above)
Comment (srpearce) [inline en_us/links/links.rst:20]: Very weird. I don't recall making either of these changes to this file.

File: en_us/shared/getting_started/dashboard_settings_profile.rst (hunk 22 above)
Comment (srpearce) [inline en_us/shared/getting_started/dashboard_settings_profile.rst]: The indications are 1) the colored bar (which does have three colors) and 2) the message. I didn't want to specify colors since colorblind learners may not be able to see them.

File: en_us/shared/getting_started/dashboard_settings_profile.rst (hunk 19 above)
Comment (srpearce) [inline en_us/shared/getting_started/dashboard_settings_profile.rst]: There's a different color for the courses that the learner has enrolled in but not completed.

File: en_us/shared/getting_started/dashboard_settings_profile.rst (hunk 18 above)
Comment (srpearce) [inline en_us/shared/getting_started/dashboard_settings_profile.rst]: A new page opens, though it's not a new browser window. I think letting the user know that the page she's on will change is important, so I'd like to leave this as-is.

**Comment by srpearce** [comment]
Addressed comments. @jsubramanian or @dabdul-hathi, can one of you review this?

File: en_us/shared/getting_started/dashboard_settings_profile.rst (hunk 22 above)
Comment (schenedx) [inline en_us/shared/getting_started/dashboard_settings_profile.rst]: With your explanation, now I get it.

File: en_us/shared/getting_started/dashboard_settings_profile.rst [hunk 23]
```diff
@@ -39,31 +38,50 @@ course or browsing the course catalog.
...
+Exploring the Dashboard
+*************************
 
-When you create an edX account or enroll in an edX course, you automatically
-go to your dashboard. You can also access your dashboard at any time by
-selecting your username in the upper right corner of any edX page.
+When you create an edX account or enroll in an edX course, you automatically go
+to your dashboard. You can also access your dashboard at any time by selecting
+your username in the upper right corner of any edX page. You can find
+information about your courses and change course settings on your dashboard.
```
Comment (jsubramanian) [inline en_us/shared/getting_started/dashboard_settings_profile.rst:48]: Should we add something about Programs here? Something like "information about your courses AND programs"

File: en_us/shared/getting_started/dashboard_settings_profile.rst [hunk 24]
```diff
@@ -39,31 +38,50 @@ course or browsing the course catalog.
...
+
+  * The **Courses** page lists all the courses you have enrolled in, whether
+    the course is current or has ended. The **Courses** page provides
+    information about your courses and allows you to change course settings. By
+    default, the dashboard opens to the **Courses** page.
+
+  * The **Programs** page lists any programs that edX offers for courses that
+    you are enrolled in. Programs appear on this page if you are enrolled in
+    any course that is part of that program. For more information, see
+    :ref:`Programs Page`.
```
Comment (jsubramanian) [inline en_us/shared/getting_started/dashboard_settings_profile.rst]: Should we say something like e.g. XSeries. Am wondering if the learner will know what a Program may be.

**Comment by jsubramanian** [comment]
Added a few comments, rest look good to me. Thanks!
